frontend/pages/admin/IntegrationsPage/cards/Vpp/VppSetupPage/VppSetupPage.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useContext, useState } from "react";
+import { InjectedRouter } from "react-router";
+
+import PATHS from "router/paths";
+import { NotificationContext } from "context/notification";
+import { getErrorReason } from "interfaces/errors";
+
+import MainContent from "components/MainContent";
+import BackLink from "components/BackLink";
+import FileUploader from "components/FileUploader";
+import DataSet from "components/DataSet";
+import Button from "components/buttons/Button";
+
+import DisableVppModal from "./components/DisableVppModal";
+import VppSetupSteps from "./components/VppSetupSteps";
+import RenewVppTokenModal from "./components/RenewVppTokenModal";
+
+const baseClass = "vpp-setup-page";
+
+interface IVppSetupContentProps {
+  router: InjectedRouter;
+}
+
+const VPPSetupContent = ({ router }: IVppSetupContentProps) => {
+  const { renderFlash } = useContext(NotificationContext);
+  const [isUploading, setIsUploading] = useState(false);
+
+  const uploadToken = async (data: FileList | null) => {
+    setIsUploading(true);
+    const token = data?.[0];
+    if (!token) {
+      setIsUploading(false);
+      renderFlash("error", "No token selected.");
+      return;
+    }
+
+    try {
+      // TODO: API integration
+      // await mdmAppleBmAPI.uploadToken(token);
+      renderFlash(
+        "success",
+        "Volume Purchasing Program (VPP) integration enabled successfully."
+      );
+      router.push(PATHS.ADMIN_INTEGRATIONS_VPP);
+    } catch (e) {
+      // TODO: error messages
+      const msg = getErrorReason(e);
+      if (msg.toLowerCase().includes("valid token")) {
+        renderFlash("error", msg);
+      } else {
+        renderFlash("error", "Couldn't Upload. Please try again.");
```

Comment:
This might be covered by your todo above, but I wanted to note the single quote styling needs to be changed.